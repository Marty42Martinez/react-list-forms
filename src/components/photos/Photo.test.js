import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';

describe('Photo Component', () => {
  it('renders a Photo', () => {
    const wrapper = shallow(<Photo photo="/real/path/to/foto" />);
    expect(wrapper).toMatchSnapshot();
  });
});
