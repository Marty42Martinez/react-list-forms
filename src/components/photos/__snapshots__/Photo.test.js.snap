// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo Component renders a Photo 1`] = `
<img
  src="/real/path/to/foto"
/>
`;
